src/tracing.rs

```diff
@@ -1,47 +1,26 @@
+#![allow(dead_code, unused_imports)] // `schema_generator` and `#[cfg(...)]` attributes produce warnings :/
```

Comment:
I don't see warnings without `unused_imports`:

```suggestion
#![allow(dead_code)] // `schema_generator` and `#[cfg(...)]` attributes produce warnings :/
```

Reply:
Did you try `cargo check --all`?

Reply:
Yeah! No warnings.

Reply:
Done.